packages/cdk/lib/cloudquery/task.ts

```diff
@@ -101,6 +101,17 @@ export interface ScheduledCloudqueryTaskProps
 	 * @see https://cloud.cloudquery.io/teams/the-guardian/api-keys
 	 */
 	cloudQueryApiKey: Secret;
+
+	/**
+	 * The image of a CloudQuery plugin that is distributed via Docker,
+	 * i.e. plugins not written in Go.
+	 *
+	 * This image will be run on its own, exposing the GRPC server on localhost:7777.
+	 * The CloudQuery source config should be configured with a registry of grpc, and path of localhost:7777.
+	 *
+	 * @see https://docs.cloudquery.io/docs/reference/source-spec
+	 */
+	dockerDistributedPluginImage?: RepositoryImage;
```

Comment:
```suggestion
	additionalPluginImage?: RepositoryImage;
```
or
```suggestion
	pluginContainer?: RepositoryImage;
```

Think this would work with any kind of Container image right? Not just Docker images?

Reply:
I used `docker` here as a way to distinguish CloudQuery plugins written in Go, vs not-Go. The former is distributed as a binary, and the latter as a Docker image, see https://docs.cloudquery.io/docs/developers/creating-new-plugin/javascript-source#releasing-and-deploying-your-plugin.